config/config.go

```diff
@@ -69,6 +69,15 @@ type (
 		CallbackProxyAPI string `ini:"callback_proxy_api"`
 	}
 
+	GitlabOauthCfg struct {
+		ClientID         string `ini:"client_id"`
+		ClientSecret     string `ini:"client_secret"`
+		ServerAddress    string `ini:"server_address"`
+		ServerName       string `ini:"server_name"`
```

Comment:
Maybe `InstanceName` / `instance_name` for this one? Or `DisplayName` / `display_name`? (I'd lean toward the latter.) _Server_ makes me think of the host / machine running Gitlab, and not of the software level or what the name is for.